fix: Add notice for missing stop name for specific location types (#1316)
MobilityData/gtfs-validator: RULES.md

| [`missing_required_column`](#missing_required_column)                                                                             | A required column is missing in the input file.                                                                                                        |
| [`missing_required_field`](#missing_required_field)                                                                               | A required field is missing.                                                                                                                           |
| [`missing_required_file`](#missing_required_file)                                                                                 | A required file is missing.                                                                                                                            |

> | [`missing_stop_name`](#missing_stop_name)                                                                                         | Missing required `stop_name` for stop with location type of `0`, `1`, or `2`.                                                                                                  |

Reviewer: ```suggestion
| [`missing_stop_name`](#missing_stop_name) | `stops.stop_name` is required for `location_type` equal to `0`, `1`, or `2`. |
```

</details>

<a name="MissingStopName"/>

### missing_stop_name

> Missing required `stop_name` for stop with location type of `0`, `1`, or `2`.

Reviewer: ```suggestion
`stops.stop_name` is required for locations that are stops (`location_type=0`), stations (`location_type=1`) or entrances/exits (`location_type=2`).
```
